Formula/zig.rb

```diff
@@ -21,16 +21,18 @@ class Zig < Formula
 
   head do
     url "https://github.com/ziglang/zig.git", branch: "master"
-    depends_on "llvm"
+    depends_on "llvm" => :build
   end
 
   depends_on "cmake" => :build
 
   fails_with gcc: "5" # LLVM is built with GCC
 
   def install
-    system "cmake", ".", *std_cmake_args, "-DZIG_STATIC_LLVM=ON"
-    system "make", "install"
+    odie "HEAD installs of `zig` are broken until ziglang/zig#12923 is resolved!" if build.head?
```

Comment:
I contemplated just removing the `head` spec, but that's bound to produce issues filed by users confused about its removal. Let's fail with an informative error message instead, since this should eventually be fixed upstream.